src/modules/auto-copy-on-click.js

```diff
@@ -0,0 +1,53 @@
+/**
+ *  @author Deux Huit Huit
+ *
+ */
+(function ($, undefined) {
+	
+	'use strict';
+
+	var site = $('#site');
+	var sels = {
+
+		toCopy: '.js-text-to-copy',
+		copyBtn: '.js-copy-on-click-btn',
+		message: '.js-confirm-message'
+	};
+	var timer = null;
+
+	var onClick = function () {
+		var t = $(this);
+		var parent = t.closest('.js-copy-ctn');
+		var message = parent.find(sels.message);
+		var delay = t.attr('data-message-delay');
```

Comment:
We should make sure to coalesce to an int here, since setTimeout only accept integers.

```js
var delay = parseInt(t.attr('data-message-delay'), 10) || 0;
```